### PR TITLE
Bugfix timeouts in api tests

### PR DESCRIPTION
**Pull request by kbeker**
Resolves https://github.com/golemfactory/concent/issues/648

### CODE REVIEW

File: concent_api/api-integration-force-accept-or-reject-results-test.py [hunk 1]
```diff
@@ -90,24 +91,28 @@ def report_computed_task(timestamp = None, task_to_compute = None):
...
 
 
-def calculate_deadline_too_far_in_the_future(current_time, concent_messaging_time):
-    return current_time - (1 + (20 * _precalculate_subtask_verification_time(concent_messaging_time)))
+def calculate_deadline_too_far_in_the_future(current_time, constants):
+    return current_time - (1 + (20 * _precalculate_subtask_verification_time(constants)))
 
 
-def _precalculate_subtask_verification_time(concent_messaging_time):
+def _precalculate_subtask_verification_time(constants):
```
Comment (rwrzesien) [inline concent_api/api-integration-force-accept-or-reject-results-test.py]: I would insist on changing this function signature to accept only required parameters.

Comment (kbeker) [reply concent_api/api-integration-force-accept-or-reject-results-test.py]: Done

File: concent_api/api-integration-force-accept-or-reject-results-test.py [hunk 2]
```diff
@@ -90,24 +91,28 @@ def report_computed_task(timestamp = None, task_to_compute = None):
...
 
-def calculate_deadline(current_time, concent_messaging_time):
-    return current_time - (1 + _precalculate_subtask_verification_time(concent_messaging_time))
+def calculate_deadline(current_time, constants):
+    return current_time - (constants.concent_messaging_time + _precalculate_subtask_verification_time(constants.concent_upload_rate, constants.concent_messaging_time))
 
 
-def calculate_deadline_too_far_in_the_future(current_time, concent_messaging_time):
-    return current_time - (1 + (20 * _precalculate_subtask_verification_time(concent_messaging_time)))
+def calculate_deadline_too_far_in_the_future(current_time, constants):
```
Comment (dybi) [inline concent_api/api-integration-force-accept-or-reject-results-test.py]: please pass `upload_rate` directly

File: concent_api/api-integration-force-accept-or-reject-results-test.py [hunk 3]
```diff
@@ -90,24 +91,28 @@ def report_computed_task(timestamp = None, task_to_compute = None):
...
+def calculate_timestamp(current_time, constants):
     return timestamp_to_isoformat(
-        current_time - (2 + _precalculate_subtask_verification_time(concent_messaging_time))
+        current_time - (2 * constants.concent_messaging_time + _precalculate_subtask_verification_time(constants.concent_upload_rate, constants.concent_messaging_time))
     )
 
 
-def calculate_deadline(current_time, concent_messaging_time):
-    return current_time - (1 + _precalculate_subtask_verification_time(concent_messaging_time))
+def calculate_deadline(current_time, constants):
```
Comment (dybi) [inline concent_api/api-integration-force-accept-or-reject-results-test.py]: please pass `concent_messaging_time` and `concent_upload_rate` directly

File: concent_api/api-integration-force-accept-or-reject-results-test.py [hunk 4]
```diff
@@ -90,24 +91,28 @@ def report_computed_task(timestamp = None, task_to_compute = None):
         )
 
 
-def calculate_timestamp(current_time, concent_messaging_time):
+def calculate_timestamp(current_time, constants):
```
Comment (dybi) [inline concent_api/api-integration-force-accept-or-reject-results-test.py]: please pass `concent_messaging_time` and `concent_upload_rate` directly

File: concent_api/api-integration-force-accept-or-reject-results-test.py (hunk 2 above)
Comment (kbeker) [reply concent_api/api-integration-force-accept-or-reject-results-test.py]: Done

File: concent_api/api-integration-force-accept-or-reject-results-test.py (hunk 3 above)
Comment (kbeker) [reply concent_api/api-integration-force-accept-or-reject-results-test.py]: Done

File: concent_api/api-integration-force-accept-or-reject-results-test.py (hunk 4 above)
Comment (kbeker) [reply concent_api/api-integration-force-accept-or-reject-results-test.py]: Done

File: concent_api/api-integration-force-accept-or-reject-results-test.py [hunk 5]
```diff
@@ -90,24 +91,28 @@ def report_computed_task(timestamp = None, task_to_compute = None):
...
 
-def calculate_deadline_too_far_in_the_future(current_time, concent_messaging_time):
-    return current_time - (1 + (20 * _precalculate_subtask_verification_time(concent_messaging_time)))
+def calculate_deadline_too_far_in_the_future(current_time, concent_upload_rate, concent_messaging_time):
+    return current_time - (1 + (20 * _precalculate_subtask_verification_time(concent_upload_rate, concent_messaging_time)))
 
 
-def _precalculate_subtask_verification_time(concent_messaging_time):
+def _precalculate_subtask_verification_time(concent_upload_rate, concent_messaging_time):
+    mdt = calculate_maximum_download_time(
```
Comment (rwrzesien) [inline concent_api/api-integration-force-accept-or-reject-results-test.py]: Please use full variables names.

Comment (kbeker) [reply concent_api/api-integration-force-accept-or-reject-results-test.py]: Done

**Pull request merged**